pkg/lightning/backend/external/engine.go

```diff
@@ -107,7 +107,7 @@ func (b *memKVsAndBuffers) build(ctx context.Context) {
 	}
 	b.droppedSizePerFile = nil
 
-	logutil.Logger(ctx).Info("building memKVsAndBuffers",
+	log.FromContext(ctx).Info("building memKVsAndBuffers",
```

Comment:
can you unify the `FromContext` of `tidb/pkg/lightning/log"` and `tidb/pkg/util/logutil"`, we can keep the latter one